client/src/components/AddBoardCard.tsx

```diff
@@ -0,0 +1,35 @@
+import React, {useContext} from 'react';
+import styled from 'styled-components';
+import { DialogContext, ModalTypes } from "../contexts/DialogContext";
+
+const AddBoardCard = ()=>{
+    const context = useContext(DialogContext);
+    return(
+        <Wrapper>
+            <AddBtn onClick={()=>{context.openModalByType(ModalTypes.CreateBoard)}}>+</AddBtn> 
+        </Wrapper>
+    );
+};
+
+export default AddBoardCard;
+
+const Wrapper = styled.div`
```

Comment:
Why do we need a Wrapper here? I think we only need 1 element for this component, no need to always wrap each component in a wrapper, it may be difficult to debug in the future.